test/support/test_helpers.ex

```diff
@@ -15,4 +17,13 @@ defmodule Benchee.TestHelpers do
         retrying(asserting_function, n - 1)
     end
   end
+
+  # assert we received eactly those messages of the contained types
+  def assert_received_exactly(expected) do
```

Comment:
This should be in `ExUnit`!! Great idea for testing these.

Reply:
We can always ask on the mailing list. Was my first thought as well, but as a rather thin wrapper I'm not sure but maybe it's worth asking?

Reply:
It's true that it's such a small thing, but it's good to have these as documented options sometimes. I know I frequently don't even think of something as a good solution to a problem I have until I stumble across it in the docs, so just having this as an available option would be really helpful for folks I think!